src/background/transactions/TransactionService.ts

```diff
@@ -41,20 +57,126 @@ export class TransactionService {
   private transactionsStore: TransactionsStore;
   private transactionsPoller: TransactionsPoller;
 
+  static ALARM_NAME = 'TransactionService:performPurgeCheck';
```

Comment:
May be it is better to name this `ALARM_NAME` more explicit?

Reply:
More explicit than `'TransactionService:performPurgeCheck'`? :)